test/tests/misc/check_error/missing_req_par_mesh_block_test.i

```diff
@@ -12,15 +12,13 @@
   nx = 3
   ny = 3
   elem_type = QUAD4
-
-  strips = 3  # this is a required params (miss-spelled) => expect to error out
```

Comment:
why remove this? looks like it tests an error

Reply:
Because the sequence of error processing/handling with the new parser is a bit different and another error in this file always triggers first.   Bla, bla, bla - it was carefully and intentionally changed :-)

Reply:
:-|